code_schemes/bungoma_s01e05.json

```diff
@@ -3,6 +3,78 @@
   "Name": "BUNGOMA S01E05",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-145fd9ba",
+      "CodeType": "Normal",
+      "DisplayText": "educate the community",
+      "NumericValue": 1,
+      "StringValue": "educate_the_community",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-cb631b67",
+      "CodeType": "Normal",
+      "DisplayText": "follow vet/doctor's prescription",
+      "NumericValue": 2,
+      "StringValue": "follow_vet_or_doctors_prescription",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-4d466ac5",
+      "CodeType": "Normal",
+      "DisplayText": "questions and narratives",
+      "NumericValue": 3,
+      "StringValue": "questions_and_narratives",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-89500909",
+      "CodeType": "Normal",
+      "DisplayText": "maintain hygiene",
+      "NumericValue": 4,
+      "StringValue": "maintain_hygiene",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-8717dbf9",
+      "CodeType": "Normal",
+      "DisplayText": "enact strict laws",
```

Comment:
this is now enact through laws in the coding frame doc

Reply:
Thanks the final codes are in the All sheet so this is okay.